Swat/SwatForm.php

```diff
@@ -476,60 +435,46 @@ public function addWithField(SwatWidget $widget, $title)
         $this->add($field);
     }
 
-    // }}}
-    // {{{ public function &getFormData()
-
     /**
-     * Returns the super-global array with this form's data
+     * Returns the super-global array with this form's data.
      *
      * Returns a reference to the super-global array containing this
      * form's data. The array is chosen based on this form's method.
      *
      * @return array a reference to the super-global array containing this
-     *                form's data.
+     *               form's data
      */
     public function &getFormData()
     {
-        $data = null;
-
-        switch ($this->method) {
-            case self::METHOD_POST:
-                $data = &$_POST;
-                break;
-            case self::METHOD_GET:
-                $data = &$_GET;
-                break;
-        }
+        match ($this->method) {
```

Comment:
Don't need to set a $data var, can just return the match expression.

Reply:
Actually, I don't think so.  At least PHPStorm is saying that only variables can be returned by reference, which means we couldn't return `null` directly from the match expression.

Reply:
Oh, you and PHPStorm are right. I don't think I've ever had to consider a by-reference function + match expression combo like this before.